torch/csrc/jit/python/init.cpp

```diff
@@ -132,170 +132,499 @@ TORCH_API void runJITCPPTests();
 #endif
 
 void initJITBindings(PyObject* module) {
-  auto m = py::handle(module).cast<py::module>();
+  auto top = py::handle(module).cast<py::module>();
+  py::module m = top.def_submodule("_jit", "Submodule containing JIT bindings");
 
   py::register_exception<JITException>(m, "JITException");
 
   py::class_<python::IODescriptor> iodescriptor(
       m, "IODescriptor"); // NOLINT(bugprone-unused-raii)
 
-  m.def("_jit_init", loadPythonClasses)
+  // Register pass and profiler functions on both torch._C and torch.C._jit
```

Comment:
nit: you can open an issue and link it here for the cleanup of this in case someone reads this code and is interested in doing it.

Reply:
I created #52720 and added a link in the comments. I'm not sure how much can be done here because we would need to find all the external callsites for these passes and fix them...